src/server/models/user.model.js

```diff
@@ -0,0 +1,22 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+// var ObjectId = Schema.ObjectId;
+
+var userSchema = new Schema({
+	quixid: Number,
+	username: {
+		type: String,
+		index: true
+	},
+	created: {
+		type: Date,
+		default: new Date()
```

Comment:
This is going to be the date at the time the schema was created, not the time the user was created.